src/datachain/cli/parser/job.py

```diff
@@ -17,7 +17,12 @@ def add_jobs_parser(subparsers, parent_parser) -> None:
     )
 
     studio_run_help = "Run a job in Studio"
-    studio_run_description = "Run a job in Studio."
+    studio_run_description = "Run a job in Studio. \n"
+    studio_run_description += (
+        "When using --start-time or --cron,"
+        " the job is scheduled as a task and will not show logs immediately."
```

Comment:
job as a task - won't be clear

just say "job is scheduled to run but won't start immediately (can be seen in the Tasks tab in UI)"